File: arrays/RemoveDuplicates.cpp
#include <bits/stdc++.h> 

int removeDuplicates(vector<int> &arr, int n) {
	// Write your code here.
	int i = 0;
	for (int j=0; j<n;j++) {
		if (arr[i] != arr[j]) {
			arr[i+1] = arr[j];
			i++;
		}
	}
	return i+1;
}

File: pythonArray/RemoveDuplicates.py
def removeDuplicates(arr,n):
    
    # i = 0;
    # for j in range(n):
    #     if arr[i] != arr[j]:
    #         arr[i+1] = arr[j]
    #         i = i+1

    return len(list(set(arr)))
    